Makefile

```diff
@@ -137,7 +137,7 @@ test-autoreview: $(PHPUNIT) vendor
 .PHONY: test-unit
 test-unit:	 	## Runs the unit tests
 test-unit: $(PHPUNIT) vendor
-	$(PHPUNIT) --group $(PHPUNIT_GROUP)
+	$(PHPUNIT) --group $(PHPUNIT_GROUP) --exclude-group e2e
```

Comment:
where do these unit tests with group `e2e` run now?

Reply:
I believe we have `make test-e2e E2E_PHPUNIT_GROUP=e2e` in `ci.yaml`:

```
      - name: Run the whole set of E2E tests
        if: runner.os != 'Windows' && matrix.e2e-runner == 'bin/infection'
        env:
          TERM: xterm-256color
        run: |
          if [[ "${{ matrix.php-version }}" == '8.2' ]]; then
            export SYMFONY_DEPRECATIONS_HELPER=max[self]=0
          fi

          make test-e2e E2E_PHPUNIT_GROUP=e2e INFECTION=bin/infection
```